transcribe.py

```diff
@@ -57,16 +61,24 @@ def main(args):
 
 		transcript_path = os.path.join(args.output_path, os.path.basename(audio_path) + '.json')
 
+		if max(ylen) > args.skip_subtitles_longer_than:
+			print(f'Too large subtitles [{ylen}] [{audio_path}]. Skipping.')
+			continue
+
 		tic = time.time()
 		y, ylen = y.to(args.device), ylen.to(args.device)
 		log_probs, olen = model(x.to(args.device), xlen.to(args.device))
 
+		if log_probs.shape[-1] > args.skip_log_probs_longer_than:
```

Comment:
This is weird. This just filters out long audio, doesn't it?

If true, then the new argument `skip_log_probs_longer_than` is not needed, just adjust `max_duration` threshold.

Reply:
I found OOM in alignment code if log_probs exceed 180k with 2 hours records and ref exceed 45k symbols. Forgot about max_duration. Try to replace it with max_duration.